### DISCUSSION TITLE
[Run issue]:DrDNASmoluchowskiReactionModel.hh:45:14: error: ‘G4double DrDNASmoluchowskiReactionModel::GetReactionRadius(const G4int&)’ marked ‘override’, but does not override

### DISCUSSION THREAD

**Discussion by nknanfeng** [topas-nbio/TOPAS-nBio#23] [Run-Problem] [answered]
### Contact Details

1569911051@qq.com

### What was the issue?

`/home/tdw/topas/build/extensions/DrDNASmoluchowskiReactionModel.hh:45:14: error: ‘G4double DrDNASmoluchowskiReactionModel::GetReactionRadius(const G4int&)’ marked ‘override’, but does not override
   45 |     G4double GetReactionRadius(const G4int&) override;
   `
Referring to the definition `GetReationRadius` in `G4VDNAReactionModel` in Geant4,shoud it be changed to the following form?

`G4double GetReactionRadius(const G4int) override;`
   

### TOPAS-nBio category

Geometry

### Version

3.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
It seems you're using TOPAS-nBio with an outdated TOPAS version. Would you confirm which TOPAS version you have?

**Reply by datboigbenga** [reply]
Hello nate, i am also facing this issue.
Kindly assisst

**Reply by nknanfeng** [reply]
Hi, By changing the version of TOPAS, I solved this problem. I hope it will be helpful to you.

**Reply by datboigbenga** [reply]
Thanks for your response.
i am using topas downloaded from the site https://www.topasmc.org/download. for macOs


Should i update to the latest version?.
if so, can you please send me the link to the verion that is compatible 

Thanks

**Reply by nknanfeng** [reply]
Because I switched to version 11.1.3 in Linux, it started working, without using the latest version.
You can refer to this link: https://github.com/OpenTOPAS/OpenTOPAS/blob/main/OpenTOPAS_quickStart_MacOS.md.
I hope it helps you.
